locksmith/src/routes/user.js

```diff
@@ -0,0 +1,10 @@
+let express = require('express')
+
+let router = express.Router()
+/* eslint-disable import/no-unresolved*/
+let userController = require('../controllers/userController')
+
+router.post('/', userController.createUser)
+router.get('/:emailAddress', userController.retrieveEncryptedPrivatekey)
```

Comment:
Could we do `/user/:emailAddress/privatekey` because I think this route is currently too generic and we may create conflicts?

Reply:
It was designed with the idea that /user/<email address> returns the relevant information that we are willing to show about the address passed. In this case the encrypted private key, the idea here being that the data returned could be expanded upon. This can be constricted to only support this specific use case

Reply:
I guess than `/user/:emailAddress` is fine, but let's not use `/:emailAddress`.

Reply:
Sorry I misunderstood how the controller prefixes the routes.